import os
names = ["base_v0\\base",
"base_v0\\base_L",
"base_v0\\caster",
"base_v0\\caster_L",
"base_v0\\pr2_wheel",
"base_v0\\wheel",
"forearm_v0\\forearm",
"forearm_v0\\wrist_flex",
"forearm_v0\\wrist_roll",
"forearm_v0\\wrist_roll_L",
"gripper_v0\\finger_tip_l",
"gripper_v0\\finger_tip_pad2_l",
"gripper_v0\\finger_tip_pad2_r",
"gripper_v0\\finger_tip_r",
"gripper_v0\\gripper_palm",
"gripper_v0\\l_finger",
"gripper_v0\\l_finger_tip",
"gripper_v0\\l_floating",
"gripper_v0\\upper_finger_l",
"gripper_v0\\upper_finger_r",
"head_v0\\head_pan",
"head_v0\\head_pan_L",
"head_v0\\head_tilt",
"head_v0\\head_tilt_L",
"sensors\\kinect2_v0\\kinect2_assembly",
"sensors\\kinect_prosilica_v0\\115x100_swept_back--coarse",
"sensors\\kinect_prosilica_v0\\115x100_swept_back_no_sensors--coarse",
"sensors\\kinect_prosilica_v0\\115x100_swept_fwd--coarse",
"sensors\\kinect_prosilica_v0\\115x100_swept_fwd_no_sensors--coarse",
"sensors\\kinect_prosilica_v0\\68-04546_Kinect_Sensor--coarse",
"sensors\\kinect_prosilica_v0\\Prosilica_w_Lens--coarse",
"sensors\\kinect_v0\\kinect_mount",
"shoulder_v0\\shoulder_lift",
"shoulder_v0\\shoulder_pan",
"shoulder_v0\\shoulder_yaw",
"shoulder_v0\\upper_arm_roll",
"shoulder_v0\\upper_arm_roll_L",
"tilting_laser_v0\\hok_tilt",
"tilting_laser_v0\\tilting_hokuyo",
"tilting_laser_v0\\tilting_hokuyo_L",
"torso_v0\\torso",
"torso_v0\\torso_lift",
"torso_v0\\torso_lift_L",
"upper_arm_v0\\elbow_flex",
"upper_arm_v0\\forearm_roll",
"upper_arm_v0\\forearm_roll_L",
"upper_arm_v0\\upper_arm"];
for x in names:
	os.system("meshlabserver -i "+x+".stl -o "+x+".obj")
	os.system("testVHACD.exe --input "+x+".obj --output "+x+".wrl --resolution=1000000 --maxNumVerticesPerCH=32")
